scipy/linalg/basic.py

```diff
@@ -105,6 +105,92 @@ def solve(a, b, sym_pos=False, lower=False, overwrite_a=False,
                      'gesv|posv' % -info)
 
 
+def solve_x(a, b, trans='N', fact='E', equed='B', af=None,
+            r=None, c=None, ipiv=None, overwrite_a=False,
+            overwrite_b=False, check_finite=True):
+    """
+    Solve the generic linear equations `a x = b` for `x` with real/complex
+    data using LU factorization where `a` is a square matrix and `x` and
+    `x` are compatible-shaped general matrices.
+
+    Error bounds on the solution and a condition estimate are also provided.
+
+    Parameters
+    ----------
+    a : (N, N) array_like
+        Square matrix
+    b : (N, NRHS) array_like
+        Right-hand side input data
+    trans : {'N', 'T', 'C'}, optional
+        Type of system to solve:
+
+        =========  ========= ==============
+        ``trans``   system    abbreviation
+        =========  ========= ==============
+          'N'      a   x = b      None
+          'T'      a^T x = b   Transpose
+          'C'      a^H x = b  Complex Conj
+        =========  ========= ==============
+    fact : {'E', 'N', 'F'}, optional
+        Selects the type of pre-conditioning on the data matrices. Specifies
+        whether or not the factorized form of the matrix ``a`` is supplied on
+        entry, and if not, whether the matrix ``a`` should be equilibrated
+        before it is factorized. See Notes for more details.
+    equed : {'B', 'N', 'R', 'C'}, optional
```

Comment:
better to use understandable variable names, not whatever LAPACK calls it. So something like ``equilibrate``. And ``factor`` or ``factorize`` above.